packages/@glimmer/runtime/test/partial-test.ts

```diff
@@ -324,6 +324,67 @@ QUnit.test('nested dynamic partial with dynamic content', () => {
   equalTokens(root, `Before <div>Testing wat are <div>Nested you doing?</div></div> After`);
 });
 
+QUnit.test('nested partial within a `{{#with}}` block', () => {
+  let template = compile(`{{#with 'Sophie' as |name|}}Before {{partial 'mantle'}} After{{/with}}`);
+
+  env.registerPartial('mantle', `[mantle {{name}} ({{age}}) {{partial 'outer'}}]`);
+  env.registerPartial('outer', `[outer {{name}} ({{age}}) {{partial 'inner'}}]`);
+  env.registerPartial('inner', `[inner {{name}} ({{age}})]`);
+
+  render(template, { age: 0 });
+
+  equalTokens(root, `Before [mantle Sophie (0) [outer Sophie (0) [inner Sophie (0)]]] After`);
+  rerender({ age: 0 }, { assertStable: true });
+  equalTokens(root, `Before [mantle Sophie (0) [outer Sophie (0) [inner Sophie (0)]]] After`);
+});
```

Comment:
This test seems superfluous now that we have the `nested partials within nested `{{#with}}` blocks` test below, let's remove it?

Reply:
Totally up to you, it certainly doesn't _hurt_ to have two tests...